scriptharness/errorlists.py

```diff
@@ -26,9 +26,9 @@ def exactly_one(key1, key2, error_check, messages):
 
       key2 (str): Dictionary key.
 
-      error_check (dict): a single item of error_list.
+      error_check (Dict[str, str or regex]): a single item of error_list.
```

Comment:
I'm not sure I'm a fan of this new style, but I'm ok landing this patch, with a few changes.